resources/js/components/entries/Listing.vue

```diff
@@ -198,8 +215,18 @@ export default {
 
             this.$axios.post(this.reorderUrl, payload)
                 .then(response => {
-                    this.$emit('reordered');
                     this.$toast.success(__('Entries successfully reordered'))
+
+                    if (this.expectsRoot) {
+                        // Reload the page so the table list SharedState.originalRows gets refreshed,
+                        // otherwise the root page will not be rendered correctly.
+                        setTimeout(() => {
+                            window.location.reload();
```

Comment:
Alternatively we could create some kind of interface with which we can update the original rows inside the shared state. I thought the UX is pretty good as is right now, since it'll only do the refresh when a root in involved. But yea, ideally you could update the original rows and just emit the `reordered` event and we would also be good. My initial thought was that it was too much effort for this use case though.